### PR TITLE
Added calling contracts

### PR DESCRIPTION
**Pull request by dylanireland**
### What does this PR fix/introduce?
Adds calling contracts to unified SDK document

### Checklist

- [x] Docs are successfully building - `yarn install && yarn run build`.
- [x] For new **internal** links I used *relative file paths* (with .md extension) - e.g. `../../faq/faq-general.md` - instead of introducing *absolute file path*, or *relative/absolute URL*.
- [x] All external links have been verified with `yarn run check:externals`.
- [x] My changes follow the [Casper docs style guidelines](https://docs.casper.network/resources/contribute-to-docs/).
- [x] All technical procedures have been tested (if you want help with this, mention it in [Reviewers](#reviewers)).

### Reviewers
@hoffmannjan 
@KMCreatesWorlds

### CODE REVIEW

File: source/docs/casper/developers/dapps/sdk/client-library-usage.md [hunk 1]
```diff
@@ -271,3 +271,74 @@ print(deploy.hash.hex())
 </Tabs>
 
 Once submitted, the above snippet will print the deploy hash in the console.
+
+---
+
+## Calling Contracts
+
+Smart contracts on the Casper Network are invoked by calling entrypoints. See below how to use Casper's SDKs to interact with these entrypoints and update the global state from a dApp:
```
Comment (ACStoneCL) [inline source/docs/casper/developers/dapps/sdk/client-library-usage.md:279]:
```suggestion
Smart contracts on a Casper network are invoked by calling entry points. See below how to use Casper's SDKs to interact with these entry points and update the global state from a dApp:
```

**Review by ACStoneCL — COMMENTED** [review]
Same as the other SDK PR - will tag @hoffmannjan for a review.

**Comment by dylanireland** [comment]
Requesting review from either @KMCreatesWorlds or @hoffmannjan

**Comment by andrzej-casper** [comment]
@ipopescu PR is now synced with `dev` branch.

**Review by KMCreatesWorlds — APPROVED** [review]
Looks good, Snippets are concise and description is sufficient from my point of view.

**Review by hoffmannjan — APPROVED** [review]
Good job!

**Pull request merged**